plugins/woocommerce/src/Admin/ReportCSVExporter.php

```diff
@@ -158,18 +158,28 @@ public function set_report_args( $args ) {
 	 * @return bool|WC_REST_Reports_Controller Report controller instance or boolean false on error.
 	 */
 	protected function map_report_controller() {
-		// @todo - Add filter to this list.
-		$controller_map = array(
-			'products'   => 'Automattic\WooCommerce\Admin\API\Reports\Products\Controller',
-			'variations' => 'Automattic\WooCommerce\Admin\API\Reports\Variations\Controller',
-			'orders'     => 'Automattic\WooCommerce\Admin\API\Reports\Orders\Controller',
-			'categories' => 'Automattic\WooCommerce\Admin\API\Reports\Categories\Controller',
-			'taxes'      => 'Automattic\WooCommerce\Admin\API\Reports\Taxes\Controller',
-			'coupons'    => 'Automattic\WooCommerce\Admin\API\Reports\Coupons\Controller',
-			'stock'      => 'Automattic\WooCommerce\Admin\API\Reports\Stock\Controller',
-			'downloads'  => 'Automattic\WooCommerce\Admin\API\Reports\Downloads\Controller',
-			'customers'  => 'Automattic\WooCommerce\Admin\API\Reports\Customers\Controller',
-			'revenue'    => 'Automattic\WooCommerce\Admin\API\Reports\Revenue\Stats\Controller',
+		/**
+		 * Used to add custom report controllers.
+		 *
+		 * @params array $controller_map - A report type to report controller class map.
+		 *
+		 * @since x.x.x
```

Comment:
Move since to the top of the docblock elements, bellow the initial comment. This way you can add future since tags on the next lines.